min_compute.yml

```diff
@@ -22,12 +22,12 @@ compute_spec:
 
     gpu:
       required: True                       # Does the application require a GPU?
-      min_vram: 8                          # Minimum GPU VRAM (GB)
+      min_vram: 18                         # Minimum GPU VRAM (GB)
```

Comment:
I think 18 is a bit too low when zephyr tries to make a big inference, at least with the current HF Pipelines, 20 allows you to mine without missing completions due to cuda OOM errors